### PR TITLE
JS Social Network own namespace declarations

### PR DESCRIPTION
**Pull request by Culprit-CZ**
Javascript declaration changed from extending jQuery to own namespace `socialNetwork`

These files are assigned to #80 and #108.
#80 contains also file `m.socialnet.js`, but the change is only by adding one property.

This property is also added to this file in this PR.

### CODE REVIEW

File: root/socialnet/js/m.activitypage.js [hunk 1]
```diff
@@ -18,50 +25,50 @@
 		_isScrollingToLoadMore: false,
 
 	    init : function(opts) {
-		    if (!$.sn._inited) { return false; }
-		    if ($.sn.enableModules.ap == undefined || !$.sn.enableModules.ap) { return false; }
-		    $.sn._settings(this, opts);
+		    if (!$sn._inited) { return false; }
```
Comment (senky) [inline root/socialnet/js/m.activitypage.js]: Please, do not use spaces - replace it with tabs. Just like it is on line 32.

Comment (Culprit-CZ) [inline root/socialnet/js/m.activitypage.js]: These spaces are there from older versions of the files.

File: root/socialnet/js/m.activitypage.js [hunk 2]
```diff
@@ -1,185 +1,197 @@
...
+
+			$('a.sn-ap-loadNews').click(function() {
+				self.loadNews();
+				return false;
+			});
+			$sn._textExpander();
+		},
+
+		/**
 		 * Nacti online usery
```
Comment (senky) [inline root/socialnet/js/m.activitypage.js]: This should be written to English:

> Load/fetch online users

File: root/socialnet/js/m.activitypage.js [hunk 3]
```diff
@@ -1,185 +1,197 @@
...
+					mode : 'onlineUsers'
+				},
+				success : function(data) {
+					$('#sn-ap .sn-ap-onlineUsers').html(data.list);
+				}
+			});
+		},
+
+		/**
 		 * Nacti nove zaznamy
```
Comment (senky) [inline root/socialnet/js/m.activitypage.js]: This should be written to English:

> Load/fetch (new entries/logs)/(news)

( I do not know that do you mean by "zaznamy", so, please, use correct word)

File: root/socialnet/js/m.socialnet.js [hunk 4]
```diff
@@ -606,30 +445,218 @@
...
+				showNewLine: true,
+				parentElement: '.sn-shareComment',
+				submitElement: 'input[name="sn-us-buttonComment"]'
+			});
+		}
+	};
+}(jQuery, socialNetwork));
+
+/**
+ * Necessary until all files are rewritten
```
Comment (senky) [inline root/socialnet/js/m.socialnet.js]: This is not description in fact :). It should look like this:

> Maps $.sn to $sn.
> 
> Necessary until all files are rewritten

File: root/socialnet/js/m.activitypage.js [hunk 5]
```diff
@@ -1,185 +1,197 @@
...
+			});
+
+			self.urlUsersAutocomplete = self.urlUsersAutocomplete.replace(/&amp;/g, '&');
+
+			$("#sn-ap-searchUsersAutocomplete").autocomplete({
+				source : self.urlUsersAutocomplete,
+				minLength : 3,
+				appendTo: '#sn-ap-searchAutocompleteContainer',
+				delay : 300,
+				select: function(event,ui){
```
Comment (senky) [inline root/socialnet/js/m.activitypage.js]: Spaces between arguments: `select: function(event, ui){`

File: root/socialnet/js/m.approval.js [hunk 6]
```diff
@@ -1,397 +1,420 @@
...
-		    	var $text = $('#sn-fms-grpCreateText');
-
-			    var gid = $.sn.getAttr($text, 'gid');
-			    var uid = $.sn.getAttr($text, 'uid');
-			    var g_t = $text.val();
-			    data = $.sn.fms._groupChange('create', gid, uid, g_t);
-			    //data = {gid:24,uid:56};
+
+/**
+ *
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: Lacks description:

> Declaration for phpBB Social Network Friends Management System module

File: root/socialnet/js/m.approval.js [hunk 7]
```diff
@@ -1,397 +1,420 @@
...
+		url : '',
+		urlFMS : '',
+		noFriends : '{ FAS FRIENDGROUP NO TOTAL }',
+		deleteUserGroup: '{ FMS_DELETE_FRIENDUSERGROUP }',
+		deleteUserGroupText: '{ FMS_DELETE_FRIENDUSERGROUP_TEXT }',
+		_init : false,
+
+		_load : function(m, s, u) {
+			var i_bl = m;
+			if (m == 'friends') i_bl = 'friend';
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: Add brackets, please:

``` javascript
if (m == 'friends') {
   i_bl = 'friend';
}
```

File: root/socialnet/js/m.approval.js [hunk 8]
```diff
@@ -1,397 +1,420 @@
...
+
+			$.ajax({
+				url : $.sn.fms.url,
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: `url: $sn.fms.url,`
(remove space after `url` and rewrite `$.sn` to `$sn`)

File: root/socialnet/js/m.approval.js [hunk 9]
```diff
@@ -1,397 +1,420 @@
...
+		},
+
+		callbackInit : function(mode) {
+			if (this._inits[mode]) {
+				return this._inits[mode].apply(this, Array.prototype.slice.call(arguments, 1));
+			} else if (typeof mode === 'object' || !mode) {
+				return mode.init.apply(this, arguments);
+			} else {
+			// $.error('Method ' + mode + ' does not exist on
+			// jQuery.sn.fms');
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: `// $sn.fms');`
even though it is comment, it should be rewritten. Or removed if unnecessary.

File: root/socialnet/js/m.approval.js [hunk 10]
```diff
@@ -1,397 +1,420 @@
...
+			self._loadFriends($('#sn-fms-groupAccordion').children('div').first());
+
+			$('#sn-fms-groupAccordion > div').droppable({
+				drop : function(event, ui) {
+					var $drag = $(this).children('div[title="' + ui.draggable.attr('title').replace(/([ #;&,.+*~\':"!^$[\]()=><|\/@])/g, '\\\\$1') + '"]');
+					if ($drag.size() > 0) {
+						return;
+					}
+
+					var i_gid = $.sn.getAttr($(this), 'gid');
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: `var i_gid = $sn.getAttr($(this), 'gid');`

File: root/socialnet/js/m.approval.js [hunk 11]
```diff
@@ -1,397 +1,420 @@
...
+					var i_uid = $.sn.getAttr(ui.draggable, 'uid');
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: `var i_uid = $sn.getAttr(ui.draggable, 'uid');`

File: root/socialnet/js/m.approval.js [hunk 12]
```diff
@@ -1,397 +1,420 @@
...
+				},
+				over : function(e, ui) {
+					sortableIn = 1;
+				},
+				out : function(e, ui) {
+					sortableIn = 0;
+				},
+				beforeStop : function(e, ui) {
+					if (sortableIn == 0) {
+
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: This blank line should be removed IMHO.

File: root/socialnet/js/m.approval.js [hunk 13]
```diff
@@ -1,397 +1,420 @@
...
+				var sub = $chld.hasClass('ui-icon-check') ? 'remove' : 'add';
+				self._groupChange(sub, gid, uid);
+				$chld.toggleClass('ui-icon-check ui-icon-no');
+				return false;
+			});
+
+			$('.sn-fms-grpCreate .ui-icon').click(function(){
+				var $text = $('#sn-fms-grpCreateText');
+
+				var gid = $.sn.getAttr($text, 'gid');
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: `var gid = $sn.getAttr($text, 'gid');`

File: root/socialnet/js/m.approval.js [hunk 14]
```diff
@@ -1,397 +1,420 @@
...
+				var uid = $.sn.getAttr($text, 'uid');
```
Comment (senky) [inline root/socialnet/js/m.approval.js]: `var uid = $sn.getAttr($text, 'uid');`

File: root/socialnet/js/m.socialnet.js [hunk 15]
```diff
@@ -1,223 +1,67 @@
...
-		_debug : false,
-		allow_load : true,
-		rtl : false,
-		expanderTextMore : '[read more]',
-		expanderTextLess : '[read less]',
-		browserOutdatedTitle : 'Your browser is outdated',
-		browserOutdated : 'Some of the features will not work on this browser, because it is outdated.',
-		showBrowserOutdated : false,
+/**
+ * Declaration phpBB Social Network object
```
Comment (senky) [inline root/socialnet/js/m.socialnet.js]: > Declaration **(for/of)** phpBB Social Network object

File: root/socialnet/js/m.socialnet.js [hunk 16]
```diff
@@ -606,30 +445,218 @@
...
+	$sn.comments = {
+		deleteTitle: 'Delete',
+		deleteText: 'Delete Text',
+		watermark: 'Watermark',
+		init: function() {
+			var self = this;
+			var confirmBox = $sn.confirmBox;
+			// Delete comment
+			$(".sn-deleteComment").live('click', function() {
+				var comment_id = $.sn.getAttr($(this), "cid");
```
Comment (senky) [inline root/socialnet/js/m.socialnet.js]: `var comment_id = $sn.getAttr($(this), "cid");`

File: root/socialnet/js/m.socialnet.js [hunk 17]
```diff
@@ -606,30 +445,218 @@
...
+				var mUrl = $.sn.getAttr($(this), 'url');
```
Comment (senky) [inline root/socialnet/js/m.socialnet.js]: `var mUrl = $sn.getAttr($(this), 'url');`

File: root/socialnet/js/m.socialnet.js (hunk 4 above)
Comment (Culprit-CZ) [inline root/socialnet/js/m.socialnet.js]: yhnx

**Comment by senky** [comment]
No, we cannot afford any problem with JS, because in case at least one problem occur, whole JS will be terminated and all sites will crash. And I really hate new PRs that only fix old PRs' problems. I will prepare testing package for testers. We need at least 3 confirmations that they did not notice any problem.

EDIT: http://phpbbsocialnetwork.com/viewtopic.php?f=18&t=4380

**Comment by Culprit-CZ** [comment]
filtered comments.

**Comment by Culprit-CZ** [comment]
Because I made squash and all comments are outdated.
All improvements will be processed after we know that the change to namespace is working correctly

File: root/socialnet/includes/sn_core_entry.php [hunk 18]
```diff
@@ -336,7 +336,7 @@ function entry_status($entry_uid, $entry_target, $entry_time)
 
 		$template->destroy_block_vars('us_status');
 
-		$comments = $this->p_master->comments->get($sn_userstatus->commentModule, 'sn-us', $entry_target, 0, 2);
+		$comments = $this->p_master->comments->get($sn_userstatus->commentModule, 'us', $entry_target, 0, 2);
```
Comment (senky) [inline root/socialnet/includes/sn_core_entry.php:341]: Why did you remove "sn-" part of the string?

Comment (Culprit-CZ) [inline root/socialnet/includes/sn_core_entry.php:341]: Much sofistics solution for comments

**Comment by Culprit-CZ** [comment]
function `_DOMChanged` is too much expensive.

**Comment by senky** [comment]
> function _DOMChanged is too much expensive.

This is, however, not a part of this PR. Please, note it somewhere, so that we can discuss it and decide on the best solution.

**Comment by Culprit-CZ** [comment]
You have right, I want only notify you

**Comment by senky** [comment]
Fine, I think we can merge it. @Culprit-CZ, if you want to squash commits, you are welcome - hit merge button by yourself.
